docs/using.adoc

```diff
@@ -0,0 +1,311 @@
+....
++
+Setting these environment variables allows programs, such as Eclipse and
+the docker CLI to access the docker daemon.
++
+[NOTE]
+====
+When the OpenShift service is running in the VM, a Docker
+registry is also started. This Docker registry can be used by external
+tools such as Eclipse to push or pull images. The Docker registry URL is
+exported as a variable, and can be accessed by running the following
+command:
+....
+$ vagrant service-manager env openshift
+# openshift env:
+# You can access the OpenShift console on: https://10.1.2.2:8443/console
+# To use OpenShift CLI, run: oc login https://10.1.2.2:8443
+export OPENSHIFT_URL=https://10.1.2.2:8443
+export OPENSHIFT_WEB_CONSOLE=https://10.1.2.2:8443/console
+export DOCKER_REGISTRY=hub.openshift.centos7-adb.10.1.2.2.xip.io
+....
+====
+
+.  Begin developing.
++
+If you do not have the docker CLI, you can use the `install-cli` command
+as:
++
+....
+$ vagrant service-manager install-cli docker
+....
++
+However, if you are using the docker CLI, you can just run it from the
+command line and it will work as expected.
++
+[NOTE]
+====
+If you encounter a Docker client and server version mismatch such as:
+....
+$ docker ps
+Error response from daemon: client is newer than server (client API version: 1.21, server API version: 1.20)
+....
+You will need to download an earlier compatible version of Docker for
+your host machine. Docker release versions and docker API versions are
+not the same. Typically, you will need to try the previous release (i.e.
+if you get this error message using a docker 1.9 CLI, try a docker 1.8
+CLI).
+====
++
+
+. If you are using Eclipse, you should follow these steps:
+..  Install the
+http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php[Docker
+Tooling] plugin.
+,.  Enable the three Docker Views (Docker Explorer, Docker Containers,
+and Docker Images) by choosing **Windows->Show Views->Others**.
+..  Enable the Console by choosing **Windows->Show Views->Console**.
+..  In the `Docker Explorer` view, click to add a connection. You should
+provide a "connection name". If your Environment Variables are set
+correctly, the remaining fields will auto-populate. If not, using the
+output from `vagrant service-manager env docker`, put the DOCKER_HOST
+variable in the "TCP Connection" field and the DOCKER_CERT_PATH in the
+"Authentication Section" Path.
+..  You can test the connection and then accept the results. At this
+point, you are ready to use ADB with Eclipse.
+
+NOTE: Testing has been done with Eclipse 4.5.0.
+
+[[using-the-box-via-ssh]]
+== Using the box via SSH
+
+Today, most users will work inside the Vagrant box. Access the box by
+using `ssh` to login to it with the following command:
+
+....
+vagrant ssh
+....
+
+You are now at a shell prompt inside the Vagrant box. You can now
+execute commands and use the tools provided.
+
+You can use the
+https://github.com/projectatomic/adb-utils/blob/master/README.rst[sccli]
```

Comment:
Considering, ADB utils will not be user facing, do we want to convert the docs in this repo too? This is the only place it is referenced and we can let it be .rst, I guess. 
Thoughts?